nautobot/dcim/filters.py

```diff
@@ -872,7 +873,7 @@ class InterfaceFilterSet(
         field_name="name",
         label="Device (name)",
     )
-    device_id = MultiValueCharFilter(
+    device_id = MultiValueUUIDFilter(
```

Comment:
Guessing this is cleanup but caused weird behavior changes with the move to `drf-spectacular`?